index.html

```diff
@@ -398,5 +398,66 @@ <h2>Transferable MediaStreamTrack</h2>
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
   </section>
+  <section>
+    <h2>VoiceIsolation constraint</h2>
+    <div>
+      <p>Some platforms offer functionality for voice isolation:
+      Attempting to remove all parts of an audio track that do not
+      correspond to a human voice. Some platforms even attempt to
+      remove extraneous voices, leaving the "main voice" as the
+        dominant component of the audio. The exact methods used may
+      vary between implementations.
+      </p>
+      <p>
+        This will have large effects on audio that is presented for
+        other reasons than to transmit voice (for instance music or
+        ambient noises), so needs to be off by default.
+      </p>
+      <p>
+        This constraint is a stronger version of noise cancellation,
+        which means that if the "noiseSuppression" constraint is set
+        to false and "voiceIsolation" is set to true, the value of
+        "noiseCancellation" will be ignored.
+      </p>
+      <p>
+        This constraint has no such relationship with any other
+        constraint; in particular it does not affect echoCancellation.
+      </p>
+    </div>
+    <div>
+      <p>
+        The WebIDL changes are the following:
+        <pre class="idl">
+partial dictionary MediaTrackSupportedConstraints {
+  boolean voiceIsolation = true;
+};
+
+partial dictionary MediaTrackConstraintSet {
+ConstrainBoolean voiceIsolation;
+};
+
+partial dictionary MediaTrackSettings {
+boolean voiceIsolation;
+};
+
+partial dictionary MediaTrackCapabilities {
+sequence&lt;boolean&gt; voiceIsolation;
+};
+        </pre>
+        </p>
+      <h4>Processing considerations</h4>
+      <p>
+        When the "voiceIsolation" setting is set to "true" by the
+        Apply Constraints algorithm, the UA
+        will attempt to remove the components of the audio track that
+        do not correspond to a human voice. If a dominant voice can be
+        identified, the UA will attempt to enhance that voice.
```

Comment:
I see my 'best effort' comment is addressed here.

Reply:
Ack.